src/util/ArrayOutput.cpp

```diff
@@ -56,7 +56,7 @@ ArrayOutput &ArrayOutput::operator+=(const ArrayOutput &rhs) {
 }
 
 size_t ArrayOutput::totalSamples() const noexcept {
-  return std::accumulate(begin(output_), end(output_), 0ULL,
+  return std::accumulate(begin(output_), end(output_), static_cast<size_t>(0ULL),
```

Comment:
Maybe just `size_t(0)` here? Or some other way to say "zero, but as a size_t. This is fine, though please drop the `ULL` if we're casting it anyway: that was only there to make it a `size_t` (but apparently not portably!!) Thanks